ScoutSuite/frontend/src/components/Breadcrumb/index.js

```diff
@@ -21,6 +23,7 @@ const Breadcrumb = () => {
         {params.service && <Service service={params.service} />}
         {params.finding && <Findings service={params.service} finding={params.finding} />}
         {params.resource && <Resources service={params.service} resource={params.resource} />}
+        {pathname.includes('/permissions/') && <Permissions service={params.service} id={params.id} />}
```

Comment:
shouldnt there also be a change for Password Policy

Reply:
added!